### DISCUSSION TITLE
Face scan without Full reindex

### DISCUSSION THREAD

**Discussion by ThrawnJL** [photoprism/photoprism#1575] [Q&A] [answered]
So I recently got Photoprism installed together with mariadb on my Synology nas.
Just after doing the full index my nas had to restart and I don’t think it got all that far with the facereq of my 30.000 or so photos.

I installed it via portainer and not via a yml file. Is it possible to run the facescan only?
I tried a docker compose command from another post by it tells me I need the yml file for that .

**Comment by lastzero** [comment, marked as answer]
Yes, see https://docs.photoprism.org/getting-started/updates/#facial-recognition

**Reply by ThrawnJL** [reply]
Thank you (again) :) 
That WAS the one I tried...which did not work because I had no YML file because of using portainer.

What I ended up doing was copy the config file from portainer and put it into a yml file in photoprism folder....and that SEEMS to work :)

**Reply by lastzero** [reply]
The CLI commands also work without Docker, just remove the prefix and start at the second photoprism.

**Reply by ThrawnJL** [reply]
Thank you  :)

I am still VERY VERY new at docked but learning slowly :)
